src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = this.header;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(slag, url) {
+    super(url);
+    this.slag = slag;
```

Comment:
The constructor of 'ArticlePage' does not modify the 'url' by adding the 'slug' at the end, as per the task description.

Reply:
fixed